src/services/notification.service.ts

```diff
@@ -9,12 +9,16 @@ interface InsertNotificationResult{
   notification_id: number;
 }
 
-const createNotification = async (notification: Notification): Promise<number> => {
+const createNotification = async (notification: Notification): Promise<unknown> => {
```

Comment:
I would use "Object" here instead of unknown, since we do know what we expect to return.

Reply:
Oh but now I see that you had Object and got an error from eslint, hmm.

Reply:
Aha, you need to create another `Interface`, as you've done for `InsertNotificationResult`.

```suggestion
interface NotificationResponse{
  notificationId: number;
  message: string;
}

const createNotification = async (notification: Notification): Promise<NotificationResponse> => {
```

Reply:
OK will do. 
I figured an interface would be an option when Object was failing, but then unknown cleared the error and thought it was ok. Will do as instructed.

Reply:
Yeah, I can see that eslint suggested `unknown`, so makes sense that you went with that!  I am not an expert as you know, but in this case we do know what we expect to get back so `unknown` feels wrong to me. :woman_shrugging: